cmd/parse.go

```diff
@@ -21,9 +21,13 @@ For example, to parse a Terraform file run the following command:
 $ snyk-iac-rules parse test.tf --format hcl2
 The '--format' flag can be left out when parsing Terraform files, as we default to hcl2.
 
-And to parse a YAML file run the following command:
+To parse a YAML file run the following command:
 $ snyk-iac-rules parse test.yaml --format yaml
 
+And to parse a Terraform plan output (in JSON format) run the following command:
```

Comment:
I guess this will be updated as per [this thread](https://snyk.slack.com/archives/C022H54L8PJ/p1638288478267400?thread_ts=1638285377.266900&cid=C022H54L8PJ)

But also could you remove the extra newline on line 30?